osrm/profiles/car-shortest-distance.lua

```diff
@@ -0,0 +1,455 @@
+-- Car profile
```

Comment:
Add more details SVP

Reply:
This is a modification of a profile found in the osrm base Docker image. Not necessary for this PR since we can use the default profile instead. Happy to remove it for now.

Reply:
Just add a few words on what this allows us to do for clarity, no need for anything fancy I agree

Reply:
I will remove this for now since I couldn't really get this profile to behave as expected (lots of timeout issues even for small service areas).